1-exercises/C-object-methods/exercise-2.js

```diff
@@ -15,8 +15,7 @@ let myPets = {
   rabbit: "Bugs",
 };
 
-let petNames =
-  /* todo */
+let petNames =/* todo */Object.values(myPets);
```

Comment:
Please check the attached link to find more methods that are stored in the Object class
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values